controllers/namespacescope_controller.go

```diff
@@ -1083,64 +1083,70 @@ func (r *NamespaceScopeReconciler) CSVReconcile(req ctrl.Request) (ctrl.Result,
 			continue
 		}
 		patchedCSVList = append(patchedCSVList, packageName.(string))
-		csv := csvList.Items[0]
-		csvOriginal := csv.DeepCopy()
-		if csv.Spec.InstallStrategy.StrategyName != "deployment" {
-			continue
-		}
-		deploymentSpecs := csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs
-		for _, deploy := range deploymentSpecs {
-			podTemplate := deploy.Spec.Template
-			// Insert restartlabel into operator pods
-			if podTemplate.Labels == nil {
-				podTemplate.Labels = make(map[string]string)
-			}
-			for k, v := range instance.Spec.RestartLabels {
-				podTemplate.Labels[k] = v
+
+		for _, csvInitial := range csvList.Items {
+			// avoid Implicit memory aliasing in for loop
+			csv := csvInitial
```

Comment:
```suggestion
		for _, c:= range csvList.Items {
			// avoid Implicit memory aliasing in for loop
			csv := c
```

csvInitial was basically a temporary variable, so may as well keep it short